src/actions/posts.js

```diff
@@ -264,31 +265,83 @@ export function getPostsAfter(channelId, postId, page = 0, perPage = Posts.POST_
 }
 
 async function getProfilesAndStatusesForPosts(list, dispatch, getState) {
-    const {currentUserId, profiles, statuses} = getState().entities.users;
+    const state = getState();
+    const {currentUserId, profiles, statuses} = state.entities.users;
     const posts = list.posts;
-    const profilesToLoad = [];
-    const statusesToLoad = [];
 
-    Object.keys(posts).forEach((key) => {
-        const post = posts[key];
+    const userIdsToLoad = new Set();
+    const statusesToLoad = new Set();
+
+    Object.keys(posts).forEach((postId) => {
```

Comment:
Oh yeah, I can change that. I forgot that existed and was just following the old pattern :p